runtime/include/flipper/types.h

```diff
@@ -32,13 +32,19 @@
 /* Weak attribute. */
 #define LF_WEAK __attribute__((weak))
 
+#ifdef __clang__
+#define LF_FUNC(MODULE) __attribute__((section("__TEXT,.lm."MODULE), used))
```

Comment:
Can you add a space between `"__TEXT,.lm."` and `MODULE`? When this header is imported in C++, this breaks, because `MODULE` is interpreted as a [literal suffix](http://en.cppreference.com/w/cpp/language/user_literal)

Reply:
Fixed in 9757f9b